### DISCUSSION TITLE
نحوه استفاده از کلودفلر در کانفیگ

### DISCUSSION THREAD

**Discussion by alireza-s13** [Gozargah/Marzban#132] [Q&A]
سلام. داخل issues یک نفر پرسیده بود که چه طور میشه از کلودفلر استفاده کرد، یک جواب کوتاه داده بودن من چند سری آزمون و خطا کردم نشد.
با فرض اینکه دامنه اصلی پنل بدون proxy هست، ممنون میشم یکی از دوستان توضیح بده که سمت کلودفلر چه کاری باید انجام بدیم؟ سمت فایل کانفیگ مرزبان کاری لازم هست انجام بشه؟

**Comment by SaintShit** [comment]
باید دامنتون رو توی کلافلر تنظیم کنید و توی داشبورد از hosts settings آیپی های کلافلر که در نظر دارین رو ست کنید با host و  sni

**Reply by Mjzaret** [reply]
دامنه رو به صورت پروکسی تنظیم کنیم یا به صورت معمولی؟ در هاست و sni دامنه خودمون که ssl گرفتیم براش رو بزاریم؟ میشه کمی بیشتر در این زمینه توضیح بدید.
تشکر
